### ISSUE TITLE
customizer JS errors after changing header margins

### ISSUE THREAD

**Issue by timelsass** [BoldGrid/boldgrid-theme-framework#157]
Steps to replicate:

1. Go to design > header >layout in the branding section change to Full Width tab.
2. save and publish, refresh customizer (went to frontend and went back to customize)
3. Go to design > header >advanced > margin - move margin top and bottom a little bit. ( i had all devices button selected [default]).
4. Go to design > header > layout in the branding section change to Container tab.

The result is the partial refresh of header fails due to js console error, and other things in the customizer no longer work (due to js errors).

- Using wp 5.3.2, latest crio from wp.org, and minfied scripts. Only crio theme active, no other plugins installed or active.

![image](https://user-images.githubusercontent.com/11907254/72224885-32d81f80-354d-11ea-958c-b1c3490b02bb.png)

console output :

```
Uncaught SyntaxError: Unexpected token '<'
front-end.min.js?ver=2.2.9:176 Uncaught TypeError: wp.customize(...) is not a function
    at Object.init (front-end.min.js?ver=2.2.9:176)
    at Object.fire (front-end.min.js?ver=2.2.9:176)
    at String.<anonymous> (front-end.min.js?ver=2.2.9:176)
    at Function.each (jquery.js?ver=1.12.4-wp:2)
    at HTMLDocument.loadEvents (front-end.min.js?ver=2.2.9:176)
    at i (jquery.js?ver=1.12.4-wp:2)
    at Object.fireWith [as resolveWith] (jquery.js?ver=1.12.4-wp:2)
    at Function.ready (jquery.js?ver=1.12.4-wp:2)
    at HTMLDocument.J (jquery.js?ver=1.12.4-wp:2)
init @ front-end.min.js?ver=2.2.9:176
fire @ front-end.min.js?ver=2.2.9:176
(anonymous) @ front-end.min.js?ver=2.2.9:176
each @ jquery.js?ver=1.12.4-wp:2
loadEvents @ front-end.min.js?ver=2.2.9:176
i @ jquery.js?ver=1.12.4-wp:2
fireWith @ jquery.js?ver=1.12.4-wp:2
ready @ jquery.js?ver=1.12.4-wp:2
J @ jquery.js?ver=1.12.4-wp:2
edit.min.js?ver=2.2.9:1 Uncaught TypeError: Cannot read property 'bind' of undefined
    at Object._customizer (edit.min.js?ver=2.2.9:1)
    at edit.min.js?ver=2.2.9:1
    at dispatch (jquery.js?ver=1.12.4-wp:3)
    at r.handle (jquery.js?ver=1.12.4-wp:3)
```

**Comment by nicolepaschen** [comment]
When I try and add padding or margin the value doesn't get saved when I close the Customizer.

![heading-padding](https://user-images.githubusercontent.com/20601160/73091674-ef64a600-3ea8-11ea-8b10-2710613d8479.gif)

**Comment by nicolepaschen** [comment]
Resolved in Crio - v2.2.11
